_source/_posts/2021-07-14-flask-scim-server.md

```diff
@@ -0,0 +1,519 @@
+
+ - *Okta Request:*
+```
+POST /scim/v2/Groups
+```
+```
+{
+    "schemas": [
+        "urn:ietf:params:scim:schemas:core:2.0:Group"
+    ],
+    "displayName": "Droids",
+    "members": [
+        {
+            "value": "cc2573a1-e1c4-491d-a3db-ae4a2e078d61",
+            "display": "bb-8@iamciam.dev"
+        },
+        {
+            "value": "478c4cf4-1aa3-41a1-93da-4c154c5955e0",
+            "display": "c-3po@iamciam.dev"
+        },
+        {
+            "value": "34e14e58-f0fe-4e9d-aeb2-0da25fa6626d",
+            "display": "r2-d2@iamciam.dev"
+        }
+    ]
+```
+ - *SCIM Server Response:*
+```
+201 CREATED
+```
+```
+{
+    "displayName": "Droids",
+    "id": "b446521a-a65b-4c0b-a5ee-0a15e8e3e908",
+    "members": [],
+    "meta": {
+        "resourceType": "Group"
+    },
+    "schemas": [
+        "urn:ietf:params:scim:schemas:core:2.0:Group"
+    ]
+}
+```
+ - *Okta Request:*
+```
+PATCH /scim/v2/Groups/b446521a-a65b-4c0b-a5ee-0a15e8e3e908
+```
+```
+{
+    "schemas": [
+        "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+    ],
+    "Operations": [
+        {
+            "op": "add",
+            "path": "members",
+            "value": [
+                {
+                    "value": "cc2573a1-e1c4-491d-a3db-ae4a2e078d61",
+                    "display": "bb-8@iamciam.dev"
+                },
+                {
+                    "value": "478c4cf4-1aa3-41a1-93da-4c154c5955e0",
+                    "display": "c-3po@iamciam.dev"
+                },
+                {
+                    "value": "34e14e58-f0fe-4e9d-aeb2-0da25fa6626d",
+                    "display": "r2-d2@iamciam.dev"
+                }
+            ]
+        }
+    ]
+}
+```
+ - *SCIM Server Response:*
+ ```
+ 200 OK
+ ```
+ ```
+ {
+    "displayName": "Droids",
+    "id": "b446521a-a65b-4c0b-a5ee-0a15e8e3e908",
+    "members": [
+        {
+            "display": "c-3po@iamciam.dev",
+            "value": "478c4cf4-1aa3-41a1-93da-4c154c5955e0"
+        },
+        {
+            "display": "bb-8@iamciam.dev",
+            "value": "cc2573a1-e1c4-491d-a3db-ae4a2e078d61"
+        },
+        {
+            "display": "r2-d2@iamciam.dev",
+            "value": "34e14e58-f0fe-4e9d-aeb2-0da25fa6626d"
+        }
+    ],
+    "meta": {
+        "resourceType": "Group"
+    },
+    "schemas": [
+        "urn:ietf:params:scim:schemas:core:2.0:Group"
+    ]
+}
+ ```
+ 
+ - And with this, you should now see your group in the scim database after running `select * from groups;`:
```

Comment:
```suggestion
And with this, you should now see your group in the scim database after running `select * from groups;`:
```

Consider removing the bullet points under the final code blocks in this example (and above).